src/HartreeParticleDSL/Particle_IR/nodes/particle_reference.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+from HartreeParticleDSL.Particle_IR.nodes.reference import Reference
+from HartreeParticleDSL.Particle_IR.nodes.member import Member
+from HartreeParticleDSL.Particle_IR.symbols.structuresymbol import StructureSymbol
+
+class ParticleReference(Reference):
```

Comment:
Double check that this makes sense to base on Reference over StructureReference? It always requires a member anyway I think?